README.md

```diff
@@ -36,3 +44,112 @@ compile 'com.imsweb.com:x12-parser:1.13'
 ```
 
 [1]: http://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.imsweb&a=x12-parser&v=LATEST
+
+An example of how to process an X12 file is shown below
+
+## Processing a file
+
+```java
+URL url = this.getClass().getResource("/837_5010/x12_example.txt");
+X12Reader reader = new X12Reader(FileType.ANSI837_5010_X222, new File(url.getFile()));
```

Comment:
I'd prefer to not use `this.getClass().getResource()`.  The constructor takes a `File` so can you just say:

```java
X12Reader reader = new X12Reader(FileType.ANSI837_5010_X222, new File("/path/file.txt"));
```